backend/src/emissions/schema.py

```diff
@@ -964,7 +969,51 @@ def mutate(root, info, input):
             return RemoveUserFromWorkingGroup(success=True)
         except ValidationError as e:
             return RemoveUserFromWorkingGroup(success=False, errors=e)
+        
+        
+class AddUserToWorkingGroup(graphene.Mutation):
+    """GraphQL mutaiton to add a user to a working group"""
     
+    class Arguments:
+        """Input strucutred defined by input type"""
+        
+        input = AddUserToWorkingGroupInput()
+        
+    success= graphene.Boolean()
+        
+    @staticmethod
+    @login_required
+    @representative_required
+    def mutate(root, info, input):
+        """Process incoming data"""
+        
+        user = info.context.user
+        
+        working_group = WorkingGroup.objects.filter(id=user.working_group.id)[0]
```

Comment:
Doesn't this throw an `IndexError` if the list of matching working groups is empty?

Reply:
Doing like it is done below for the `user_to_add_query_set` wouldn't raise one.